src/client/Client.js

```diff
@@ -414,6 +415,14 @@ class Client extends BaseClient {
       query.set('guild_id', guildID);
     }
 
+    if (options.additionalScopes) {
+      const scopes = options.additionalScopes;
+      if (!Array.isArray(scopes) || scopes.some(scope => !InviteScopes.includes(scope))) {
+        throw new TypeError('INVALID_TYPE', 'additionalScopes', 'Array of Invite Scopes', true);
+      }
```

Comment:
Do we not want to throw an error with the invalid scope in question?

Reply:
In general, the library does not emit the cause of an error when checking within Arrays / Collections (erroring with similar messages as seen here). I would like to change the INVALID_TYPE error to be able to include a more descriptive error when dealing with said Arrays / Collections in general, but that is outside the scope of this PR.

If you think this should use a different error type that would solve this problem, let me know.

Reply:
In fact #5228 implemented the problematic bit in BitField, this is more a suggestion to adopt features we started implementing with v13, rather than keeping with what we did in the previous versions (v12 and lower).

Reply:
I see, I would think this would need a general refactor (this PR can add the error) to use `PARAMETER_INVALID` or something similar and perform the checks in different steps. I would love this overall to help neatly narrow down where errors occurred